packages/grafana-ui/src/components/VizTooltip/VizTooltipFooter.tsx

```diff
@@ -88,6 +89,13 @@ export const VizTooltipFooter = ({ dataLinks, actions = [], annotate }: VizToolt
     <div className={styles.wrapper}>
       {!hasOneClickAction && renderDataLinks(dataLinks, styles)}
       {!hasOneClickLink && renderActions(actions, styles)}
+      {!hasOneClickLink && !hasOneClickAction && onFilterClick && (
```

Comment:
There's some logic here which is untested. I wanted to see how the "Add annotation" button (and the logic that controls its inclusion) was tested, so I removed it to see which tests fail. No tests failed, meaning it has no tests! See this PR's CI run.
So I do plan to investigate adding tests both for this and for the "Add annotation" button